content/docs/03-ai-sdk-core/15-tools-and-tool-calling.mdx

```diff
@@ -624,6 +624,87 @@ export const weatherTool = tool({
 });
 ```
 
+## MCP Tools
+
+The AI SDK supports connecting to Model Context Protocol (MCP) servers to access their tools. This enables your AI applications to discover and use tools across various services through a standardized interface.
+
+### Using MCP Tools
+
+Create an MCP client using either `stdio` (for local servers) or `SSE` (for web servers):
```

Comment:
a little explanation would stdio vs sse is could be nice (1 sentence each maybe)